xen/include/public/io/displif.h

```diff
@@ -0,0 +1,620 @@
+/******************************************************************************
+ * displif.h
+ *
+ * Unified display device I/O interface for Xen guest OSes.
```

Comment:
I guess this needs to be extended to better explain what this driver is and, more importantly, what it is not...